packages/create-fuels/src/lib/getPackageManager.test.ts

```diff
@@ -54,32 +51,22 @@ describe('getPackageManager', () => {
   it.each(runScenarios)(
     'should have the correct run commands',
     (packageManager, expectedRunCommand) => {
-      const command = getPackageManager({ [packageManager]: true });
+      process.env.npm_config_user_agent = packageManager;
+
+      const command = getPackageManager();
 
       const run = command.run(runCommand);
 
       expect(run).toEqual(expectedRunCommand);
     }
   );
 
-  it('should warn the user if more than one package manager selected', () => {
-    const { warn } = mockAllDeps();
-    const opts = { pnpm: true, npm: true };
-
-    getPackageManager(opts);
-
-    expect(warn).toBeCalledWith('More than one package manager was selected.');
-  });
-
-  it('should default to npm if no package manager is selected', () => {
+  it('should default to npm', () => {
     const packageManager = 'npm';
     const expectedPackageManager = packageMangers[packageManager];
-    const { warn } = mockAllDeps();
```

Comment:
Why remove the mocking here?

Could we not assert that the expected warning is emitted? It also stops the noise in the test outputs ([ref](https://github.com/FuelLabs/fuels-ts/actions/runs/12517820318/job/34919255843?pr=3503#step:6:1113))